src/components/MoneyTemporaryForRefactorRequestConfirmationList.tsx

```diff
@@ -200,37 +200,41 @@ function MoneyTemporaryForRefactorRequestConfirmationList({
     iouComment,
     receiptFilename = '',
     listStyles,
-    mileageRates,
-    onConfirm,
-    onSelectParticipant,
-    onSendMoney,
+    iouCreated,
+    iouIsBillable = false,
     onToggleBillable,
     hasSmartScanFailed,
     reportActionID,
 }: MoneyRequestConfirmationListProps) {
     const theme = useTheme();
     const styles = useThemeStyles();
     const {translate, toLocaleDigit} = useLocalize();
-    const {canUseP2PDistanceRequests, canUseViolations} = usePermissions(iouType);
+    const currentUserPersonalDetails = useCurrentUserPersonalDetails();
+    const {canUseP2PDistanceRequests, canUseViolations} = usePermissions();
```

Comment:
I think this still needs to take in the iouType

Reply:
done!